providers/src/airflow/providers/oracle/hooks/oracle.py

```diff
@@ -350,11 +354,23 @@ def bulk_insert_rows(
             self.set_autocommit(conn, False)
         cursor = conn.cursor()  # type: ignore[attr-defined]
         values_base = target_fields or rows[0]
-        prepared_stm = "insert into {tablename} {columns} values ({values})".format(
-            tablename=table,
-            columns="({})".format(", ".join(target_fields)) if target_fields else "",
-            values=", ".join(f":{i}" for i in range(1, len(values_base) + 1)),
-        )
+
+        if sequence_column and sequence_name:
```

Comment:
What happens when only sequence_column but no seuence_name (or the other way round) are specified? I think it should be treated as error. You either need none or both.

Reply:
That's right.
Currently, an error occurs even if you do not enter either of them.
I'll add a code for that case. Thank you